src/test/groovy/org/prebid/server/functional/CcpaSpec.groovy

```diff
@@ -44,8 +48,8 @@ class CcpaSpec extends BaseSpec {
     def "PBS should not mask publisher info when privacy.ccpa.enabled = false in account config"() {
         given: "Default basic generic BidRequest"
         def bidRequest = BidRequest.defaultBidRequest
-        def valid_ccpa = "1YYY"
-        bidRequest.regs.ext = new RegsExt(gdpr: 0, usPrivacy: valid_ccpa)
+        def validCcpa = "1YYY"
```

Comment:
While you're at it, you can also create a wrapper class for it, because a lot of people won't know what 1YYY means.